supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -27,6 +28,13 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
 
   private implicit val ec = context.dispatcher
   private val maxRetries = 10
+  // FIXME: Determined empirically that we start to get errors when the batch request approaches 15MB.
+  // Find out what the real value is
+  private val maxBatchRequestSize = 14 * 1024 * 1024
+  private val requestTooLargeException = new JesApiException(new IllegalArgumentException(
+    "The JES creation request exceeds the maximum size allowed. " +
+      "If you have a task with a very large number of inputs and / or outputs in your workflow you should try to reduce it."
```

Comment:
> Try bundling your inputs into a zip file rather than submitting individually

Oh I like this, if users have a problem Cromwell just tells them to zip it.